cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/AzureManagementCluster.constants.tsx

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-unused-vars */
 export enum AZURE_FIELDS {
```

Comment:
Not important but const enum could work here? :)

Reply:
yes good catch